math/genvector/inc/Math/GenVector/Transform3D.h

```diff
@@ -606,36 +868,96 @@ class Transform3D {
    /**
       make transformation from first a rotation then a translation
    */
-   void  AssignFrom( const Rotation3D & r, const Vector & v);
+   void AssignFrom( const Rotation3D & r, const Vector & v)
+   {
+     // assignment  from rotation + translation
+     
+     T rotData[9];
+     r.GetComponents(rotData, rotData +9);
+     // first raw
+     for (int i = 0; i < 3; ++i)
+       fM[i] = rotData[i];
+     // second raw
+     for (int i = 0; i < 3; ++i)
+       fM[kYX+i] = rotData[3+i];
+     // third raw
+     for (int i = 0; i < 3; ++i)
+       fM[kZX+i] = rotData[6+i];
+     
+     // translation data
+     T vecData[3];
+     v.GetCoordinates(vecData, vecData+3);
+     fM[kDX] = vecData[0];
+     fM[kDY] = vecData[1];
+     fM[kDZ] = vecData[2];
+   }
+
 
    /**
       make transformation from only rotations (zero translation)
    */
-   void  AssignFrom( const Rotation3D & r);
+   void  AssignFrom( const Rotation3D & r)
```

Comment:
This is all I get from that command

```
~/Projects/ROOT/source > git-clang-format-mp-3.9 HEAD~1 --diff
diff --git a/math/mathmore/inc/Math/MCParameters.h b/math/mathmore/inc/Math/MCParameters.h
index e4a8baab20..98e5dd66c2 100644
--- a/math/mathmore/inc/Math/MCParameters.h
+++ b/math/mathmore/inc/Math/MCParameters.h
@@ -26,7 +26,7 @@
 #ifndef ROOT_Math_MCParameters
 #define ROOT_Math_MCParameters
 
-#include <stddef.h>   // for size_t
+#include <stddef.h> // for size_t
 
 namespace ROOT {
 namespace Math {
diff --git a/math/physics/src/TVector3.cxx b/math/physics/src/TVector3.cxx
index 8bd5e12513..6d7a47ad39 100644
--- a/math/physics/src/TVector3.cxx
+++ b/math/physics/src/TVector3.cxx
@@ -166,7 +166,6 @@ the theta plane and y' in the xy plane as well as perpendicular to the
 theta plane) to the (x,y,z) frame.
 */
 
-
 #include "TMatrix.h"
 #include "TVector3.h"
```

Thats all. A lot less than running directly on the files. So what should I do here ?